rollbar/resource_project.go

```diff
@@ -192,11 +218,20 @@ func resourceProjectRead(ctx context.Context, d *schema.ResourceData, m interfac
 	var mProj map[string]interface{}
 	mustDecodeMapStructure(proj, &mProj)
 	for k, v := range mProj {
-		if k == "id" {
+		if k == "id" || k == settingsData {
 			continue
 		}
 		mustSet(d, k, v)
 	}
+
+	for k, v := range mProj[settingsData].(map[string]interface{}) {
+		if (k == projectTimezone && v == timezoneDefault) ||
```

Comment:
Could you please add a brief documentation on what problem this piece of code is solving? It's going to fade from memory quickly.

Reply:
this is for backward compatibility, we don't want to update many projects for a user - that would create a lot of confusion. I will add some comment

Reply:
added